server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -347,11 +347,6 @@ public ServerEnvironment(final String hostControllerName, final Properties props
         javaExtDirs = getFilesFromProperty(JAVA_EXT_DIRS, props);
 
         if (launchType.equals(LaunchType.SELF_CONTAINED)) {
-            Path[] supplementalConfigurationFiles = findSupplementalConfigurationFiles(null, supplementalConfiguration);
-            ConfigurationExtension configurationExtension = ConfigurationExtensionFactory.createConfigurationExtension(supplementalConfigurationFiles);
-            if (configurationExtension != null) {
-                configInteractionPolicy = configurationExtension.shouldProcessOperations(runningModeControl) ? ConfigurationFile.InteractionPolicy.READ_ONLY : configInteractionPolicy;
```

Comment:
Note to reviewers:

The configInteractionPolicy var is never read in this branch, making the assignment here pointless and thus the other removed lines pointless as well.

There are uses of the var in other branches.